### ISSUE TITLE
Current: `{{ container_name`}}	 is not expanded on grafana dash bord.

### ISSUE THREAD

**Issue by terrada3** [coreos/prometheus-operator#1076]
**What did you do?**
install  kube-prometheus and prometheus-operator by using helm

**What did you expect to see?**
Container names are shown in pods dashbord.

**What did you see instead? Under which circumstances?**
A variable that is container_name is not expanded actual value.

```
Current: `{{ container_name`}}	
```

Other variables are not expanded too.
![image](https://user-images.githubusercontent.com/4397382/37334491-5e0ce87a-26ef-11e8-9b06-1a7d7211582f.png)

**Environment**

* Kubernetes version information:
  ```
  Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.4", GitCommit:"793658f2d7ca7f064d2bdf606519f9fe1229c381", GitTreeState:"clean", BuildDate:"2017-08-17T08:48:23Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
  Server Version: version.Info{Major:"1", Minor:"8+", GitVersion:"v1.8.3-rancher1", GitCommit:"beb8311a9f114ba92558d8d771a81b7fb38422ae", GitTreeState:"clean", BuildDate:"2017-11-14T00:54:19Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
  ```
* Kubernetes cluster kind: 
Rancher Kubernetes Engine

* Manifests:
Using helm command.
  ```
  helm install coreos/kube-prometheus --name kube-prometheus --set global.rbacEnable=true --
  namespace monitoring
  helm install prometheus-operator --name prometheus-operator --namespace=monitoring
  ```
  ```
  NAME                    REVISION        UPDATED                         STATUS          CHART                         NAMESPACE
  kube-prometheus         1               Tue Mar 13 16:20:23 2018        DEPLOYED        kube-prometheus- 0.0.29       monitoring
  prometheus-operator     1               Tue Mar 13 17:46:05 2018        DEPLOYED        prometheus-operator-0.0.15    monitoring
  ```

* Prometheus Operator Logs:

```
insert Prometheus Operator logs relevant to the issue here
```

**Comment by gianrubio** [comment]
This is a know issue,  introduced on #1006. @vglafirov do you have time to work on this issue?

**Comment by vglafirov** [comment]
@gianrubio #1081 should fix this issue.

**Comment by terrada3** [comment]
@gianrubi, @vglafirov 
Thank you so much. And sorry for my slow reply.

**Comment by gianrubio** [comment]
Fixed by #1081
